tests/flow/tests_pytorch.py

```diff
@@ -928,3 +929,26 @@ def test_pytorch_model_rdb_save_load(env):
     env.assertTrue(dtype_memory == dtype_after_rdbload)
     env.assertTrue(shape_memory == shape_after_rdbload)
     env.assertTrue(data_memory == data_after_rdbload)
+
+
+def test_parallelism():
+    env = Env(moduleArgs='INTRA_OP_PARALLELISM 1 INTER_OP_PARALLELISM 1')
```

Comment:
@hhsecond LGTM, but while were at it can we add @DvirDukhan recomendation on the check:
```
res = r.executeCommand('INFO MODULES')
## check for the ai section
## check for the inter and intra op values there
env.assertEqual(3, res[index_of_inter_op_config])
```

Reply:
@hhsecond agree with @filipecosta90

Reply:
Done